muser/src/main/java/edu/usf/cutr/mfe/manager/MuserMusicDataAnalysisManager.java

```diff
@@ -70,6 +72,45 @@ private void processUserById(String userId) {
 			        tbr1.setmYear(tbr.song.getYear());
 			        musicAnalysisList.add(tbr1);
 			        mCSVFileWriter.appendAllToCsV(musicAnalysisList);
+			        musicAnalysisList.clear();
+			       
+	        }
+	}
```

Comment:
An extra line on 76 and please give an extra line between 2 function definitions.